ansible/group_vars/openwrt/main.yml

```diff
@@ -8,7 +8,6 @@ openwrt_packages_remove:
 openwrt_packages_add:
   - haveged
   - htop
-  - hwclock
```

Comment:
~Why remove `hwclock`?~ I guess the APs do not have hardware clocks?

We can keep `hwclock` in the default, and put `"-hwclock"` in the accesspoint's extras.
OTOH, I'm not sure which option is the right default there.

Reply:
I think most routers and other stuff that usually runs OpenWRT don't have hardware clocks and if they have one they are mostly not battery backed. Also given the fact that hardware clocks in Linux are only ever used/updated during boot and shutdown it doesn't make sense for most hardware to have hwclock support at all.

Reply:
Yeah, fair.